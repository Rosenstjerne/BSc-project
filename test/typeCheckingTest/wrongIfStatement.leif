if (5) print(5);