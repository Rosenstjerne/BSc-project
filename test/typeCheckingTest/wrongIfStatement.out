
Error in phase Type Checking, line 1:
    An expression of type 'int' cannot be used as a guarde. Must be of type 'bool'
